syntax = "proto3";

package aircipher;

service AirService {
  rpc SendMessage(Message) returns (MessageAck);
}

message Message {
  string sender_uid = 1;
  string reciever_uid = 2;
  string payload = 3;
  int64 timestamp = 4;
}
message MessageAck {
  bool recieved = 1;
  string status = 2;
}
